src/backend/model/repositories/shop.model.ts

```diff
@@ -47,3 +47,37 @@ export async function findShopById(id: number): Promise<modelType.ShopGet> {
 
   return shop;
 }
+
+/**
+ * Get all cities shop data.
+ *
+ * @returns Shop data find by id
+ */
+export async function getCities(): Promise<modelType.CitiesGet> {
+  let cities: modelType.CitiesGet;
+
+  try {
+    cities = await knex.select("city").from(TABLE_SHOPS);
+  } catch (error) {
+    throw error;
+  }
+
+  return cities;
+}
+
+/**
+ * Get all shop data.
+ *
+ * @returns Shop data find by id
+ */
+export async function getAllShops(): Promise<modelType.AllShopGet> {
+  let allShops: modelType.AllShopGet;
+
+  try {
+    allShops = await knex.select("*").from(TABLE_SHOPS);
```

Comment:
Specifying `*` to get the all columns might be a little problem. Because we want the fields that specified in the  `modelType.AllShopGet`. We don't want `updated_at` and `created_at`. Columns should always be clearly specified to reduce data volume as much as possible. It is also clearer if you write in the `select` which columns to get.

Reply:
Also add a `order by`.